marlowe-dashboard-client/src/Marlowe/Execution/State.purs

```diff
@@ -136,7 +78,7 @@ timeoutState currentSlot { current, previous, mPendingTimeouts, mNextTimeout } =
     advanceAllTimeouts ::
       Maybe Slot ->
       Array Slot ->
-      State ->
+      Marlowe.State ->
```

Comment:
once again, the `Marlowe` prefix does not clarify which state is this referring to